custom-fonts.md

```diff
@@ -1,13 +1,19 @@
 ---
 layout: page
-title: Using Custom Fonts
+title: Using Fonts
 
 permalink: /custom-fonts/
 ---
 
-You can use custom fonts in your Flutter application by including them in your
-`pubspec.yaml` file under the `fonts` heading in your `flutter` section:
+## Using fonts
```

Comment:
I noticed an introduction on the other guides. Maybe it's better suited to keep it consistent, I'll try to revise it to better fit.